### PR TITLE
Update licenses with a little more detail

### PR DESCRIPTION
**Pull request by lee-dohm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,10 @@ Content is released under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.
 > <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY-4.0</a>
 > license.</a>
 
-Code, including source files and code samples if any in the content, is released under [CC0-1.0](https://creativecommons.org/publicdomain/zero/1.0/).
+Code, including source files and code samples if any in the content, is released under [CC0-1.0](https://creativecommons.org/publicdomain/zero/1.0/), with the following exceptions:
+
+* The primer components in the `node_modules` directory are under the [MIT license](https://github.com/primer/octicons/blob/master/LICENSE)
```
Comment (bkeepers) [inline README.md]: https://github.com/primer/primer-css/blob/master/LICENSE

??

Comment (lee-dohm) [inline README.md]: I don't know where the canonical "MIT license" lives, so I copied the link from the Primer README :grinning:

Comment (lee-dohm) [inline README.md]: Should I point it to https://opensource.org/licenses/MIT instead?

Comment (mlinksva) [inline README.md]: I wouldn't link to any copy of the license.  The key is that there is a copy of MIT license of each primer component in their directories.

**Pull request merged**